topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -17,7 +17,6 @@ key_points:
 - Pharmacophore-based target prediction is an efficient and cost-effective method.
 contributors:
 - aurelienmoumbock
-- simonbray
```

Comment:
@aurelienmoumbock thanks for adding me here, but my contribution was pretty small. I would rather not be listed as a contributor, if that's ok for you.

Reply:
@simonbray I disagree 😐 Without your help I would not have figured out how to install RDconf or create the tutorial. Moreover, you polished the tutorial.

Reply:
Okay, then I will readd myself :)